pages/mesosphere/dcos/services/hdfs/2.6.0-3.2.0/release-notes/index.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes
+excerpt: Discover the new features, updates, and known limitations in this release of the HDFS Service
+title: Release Notes
+menuWeight: 0
+model: /mesosphere/dcos/services/hdfs/data.yml
+render: mustache
+---
+
+# Release Notes for HDFS Service version 2.6.0-3.2.0
+
+## Feature
+- HDFS cluster can be accessed from outside the DC/OS cluster by setting up an external DNS. This feature can be utilized through `custom_domain` property.
```

Comment:
```suggestion
- Added support for custom domains. See <add link to custom domain section here>
- Added support for DC/OS Storage Service (DSS). See [DSS documentation](https://docs.d2iq.com/mesosphere/dcos/services/storage/1.0.0/)
- Added marathon service scheduler checks
```

Reply:
Please add a section in HDFS docs (could be security) similar to this: https://docs.d2iq.com/mesosphere/dcos/services/cassandra/2.6.0-3.11.4/security/#forwarding-dns-and-custom-domain

That way users can reference what to do with custom domains

Reply:
Done